ktor-client/ktor-client-core/ktor-client-core-js/src/io/ktor/client/engine/js/JsUtils.kt

```diff
@@ -36,7 +36,7 @@ internal suspend fun CoroutineScope.toRaw(clientRequest: HttpRequest): RequestIn
 }
 
 internal suspend fun fetch(url: Url, request: RequestInit): Response = suspendCancellableCoroutine {
-    window.fetch(url.toString(), request).then({ response ->
+    Utils.get().fetch(url.toString(), request).then({ response ->
```

Comment:
Let's add `fun Utils(): Utils` factory method and store utils instance in the client engine.